pkg/controller/velerobackup/velerobackup_controller.go

```diff
@@ -129,6 +146,22 @@ func (r *ReconcileBackup) Reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, err
 	}
 
+	// Check to see how long since the last back was taken (we may need to rate limit)
+	delta := metav1.Now().Sub(cp.Spec.LastBackupTime.Time)
+	if delta < reconcileRateLimitDuration {
+		// calculate the next time this reconcile should attempt to run. Add 1 second to avoid a race condition right
+		// on the boundary of when this will be outside of the rate limit.
+		requestAfter := (reconcileRateLimitDuration - delta) + (1 * time.Second)
+
+		nsLogger.Infof("Rate limiting this reconcile. Will reconcile this namespace again in %v", requestAfter)
+
+		// Requeue this reconcile because we've already taken a backup within the rate limit duration.
+		return reconcile.Result{
+			Requeue:      true,
```

Comment:
The `Requeue` field does not need to be set to `true`. Setting `RequeueAfter` is sufficient.